lib/route/link/can.c

```diff
@@ -864,6 +893,30 @@ int rtnl_link_can_set_data_bittiming(struct rtnl_link *link,
 	return 0;
 }
 
+/**
+ * Get CAN device stats
+ * @arg link            Link object
+ * @arg device_stats	CAN device stats
+ *
+ * @return 0 on success or a negative error code
+ */
+int rtnl_link_can_get_device_stats(struct rtnl_link* link,
+				   struct can_device_stats *device_stats)
+{
+	struct can_info *ci = link->l_info;
+
+	IS_CAN_LINK_ASSERT(link);
+	if (!device_stats)
+		return -NLE_INVAL;
+
+	if (ci->ci_mask & CAN_HAS_DEVICE_STATS)
+		*device_stats = ci->ci_device_stats;
+	else
+		return -NLE_AGAIN;
```

Comment:
`return -NLE_MISSING_ATTR` ?

Reply:
NLE_MISSING_ATTR seems like a better error here. However,  all other getters in this file also return NLE_AGAIN when the attribute is absent. Should the other getters also be changed to NLE_MISSING_ATTR?